src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1403,12 +1407,16 @@ expert-ds:experts
 user:users
 ----
 
-When defined via <<_ldap,LDAP>>, the name of the <<_configure_tenant_attribute,LDAP attribute>> which will contain tenant information must be configured.
+When defined as a <<_ldap,LDAP attribute>>, the name of the <<_configure_tenant_attribute,attribute>> which will contain tenant information must be configured.
+
+When the authentication method is set to <<_ldap_with_multiple_domains,LDAP with Multiple Domains>>, the tenant is automatically set to the current user _LDAP domain_.
 
-Tenants are used to isolate groups of users, where users belonging to one tenant will not see the activities launched by users belonging to other tenants.
+Tenants can be used to isolate groups of users, where users belonging to one tenant will not see the activities launched by users belonging to other tenants.
```

Comment:
Tenants allow users to work in isolated silos, where users belonging to one tenant **cannot** not see the jobs of users belonging to other tenants.